Mini-lab Java Refresher 1-Simple Banking App/src/tests/UserTest.java

```diff
@@ -0,0 +1,64 @@
+package tests;
+import model.User;
+import utils.TestUtils;
+
+public class UserTest {
+    public static void main(String[] args) {
+        testUserConstructor();
+    }
+
+    public static void testUserConstructor() {
+        String testUsername = "kenny";
+        String testPassword = "Password";
+        String testFirstName = "Kenton";
+        String testLastName = "McComb";
+        String testMobileNumber= "07712239587";
+
+        // 2 run the object under test
+        User testUser = new User (testUsername, testPassword, testFirstName, testLastName, testMobileNumber);
```

Comment:
The brackets for a constructor should be attached to the data type (User) without spaces:
```suggestion
        User testUser = new User(testUsername, testPassword, testFirstName, testLastName, testMobileNumber);
```

Reply:
Removed unnecessary whitespace after new User.